docs/src/main/tut/dsl.md

```diff
@@ -242,35 +244,35 @@ in its HTTP envelope, and thus has what it needs to calculate a
 
 #### Streaming bodies
 
-Streaming bodies are supported by returning a `scalaz.stream.Process`.
+Streaming bodies are supported by returning a `fs2.Stream`.
 Like `Future`s and `Task`s, the stream may be of any type that has an
 `EntityEncoder`.
 
-An intro to scalaz-stream is out of scope, but we can glimpse the
+An intro to `Stream` is out of scope, but we can glimpse the
 power here.  This stream emits the elapsed time every 100 milliseconds
 for one second:
 
 ```tut:book
+implicit val scheduler = fs2.Scheduler.fromFixedDaemonPool(2, threadName = "scheduler")
 val drip = {
   import scala.concurrent.duration._
-  implicit def defaultScheduler = scalaz.concurrent.Strategy.DefaultTimeoutScheduler
-  scalaz.stream.time.awakeEvery(100.millis).map(_.toString).take(10)
+  fs2.time.awakeEvery[Task](100.millis).map(_.toString).take(10)
 }
 ```
 
 We can see it for ourselves in the REPL:
 
 ```tut
-drip.to(scalaz.stream.io.stdOutLines).run.unsafePerformSync
+drip.through(fs2.text.lines).through(_.evalMap(s => {Task.delay{println(s); s}})).run.unsafeRun
```

Comment:
stdOutLines does not exist in fs2. This is the best I came up with, based on something I found via googling. Is there something better?